File: python/mxnet/_global_var.py
_ndarray_cls = None
_np_ndarray_cls = None

def _set_ndarray_class(cls):
    global _ndarray_cls
    _ndarray_cls = cls


def _set_np_ndarray_class(cls):
    global _np_ndarray_cls
    _np_ndarray_cls = cls

File: csharp-package/src/MxNet/GlobalVar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MxNet
{
    public class GlobalVar
    {
        public static Type _ndarray_cls = null;

        public static Type _np_ndarray_cls = null;

        public static void SetNDArrayClass(Type cls)
        {
            _ndarray_cls = cls;
        }

        public static void SetNumpyNDArrayClass(Type cls)
        {
            _np_ndarray_cls = cls;
        }
    }
}
